.github/workflows/pre-release.yaml

```diff
@@ -0,0 +1,44 @@
+name: Pre-release
+
+on:
+  pull_request:
+    branches: [ "master" ]
+  workflow_dispatch:
+
+jobs:
+  wallet-e2e-tests:
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04 ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: 📥 Checkout repository
+        uses: actions/checkout@v2.3.3
+        with:
+          submodules: true
+
+      - name: 🧰 Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.17
+
+      - name: 🔨 Install & Build
+        run: |
+          yarn install --frozen-lockfile --non-interactive --logevel=error
+          yarn build
+
+      - name: 🔬 Test
+        working-directory: packages/wallet
+        env:
+          BLOCKFROST_API_KEY: ${{ secrets.BLOCKFROST_API_KEY }}
+          MNEMONIC_WORDS_ONE: ${{ secrets.MNEMONIC_WORDS_ONE }}
+          MNEMONIC_WORDS_TWO: ${{ secrets.MNEMONIC_WORDS_TWO }}
```

Comment:
We only want `MNEMONIC_WORDS`

Reply:
Also seems like we're missing some variable(s) in env because the check on this PR failed